content/de/how-it-works.md

```diff
@@ -8,57 +8,59 @@ lastmod: 2019-10-18
 
 {{< lastmod >}}
 
-Das Ziel von Let's Encrypt und des [ACME-Protokolls](https://tools.ietf.org/html/rfc8555) besteht darin, die Einrichtung eines HTTPS-Servers zu ermöglichen, der automatisch ein vertrauenswürdiges Browserzertifikat ohne menschliches Eingreifen erhält. Dies wird durch Ausführen eines Zertifikatsverwaltungsagenten auf dem Webserver erreicht.
+Das Ziel von Let's&nbsp;Encrypt und des [ACME-Protokolls](https://tools.ietf.org/html/rfc8555) besteht darin, die Einrichtung eines HTTPS-Servers zu ermöglichen, der automatisch ein vertrauenswürdiges Browserzertifikat ohne menschliches Eingreifen erhält.  Dies wird durch Ausführen eines Zertifikatsverwaltungsagenten auf dem Webserver erreicht.
 
-Um zu verstehen, wie die Technologie funktioniert, gehen wir die Einrichtung von `https://example.com/` mit einem Zertifikatsverwaltungsagenten durch, der Let's Encrypt unterstützt.
-
-Dieser Prozess umfasst zwei Schritte. Zunächst weist der Agent der Zertifizierungsstelle nach, dass der Webserver eine Domain kontrolliert. Anschliessend kann der Agent Zertifikate für diese Domain anfordern, erneuern und widerrufen.
+Um zu verstehen, wie die Technologie funktioniert, gehen wir die Einrichtung von `https://example.com/` mit einem Zertifikatsverwaltungsagenten durch, der Let's&nbsp;Encrypt unterstützt.
```

Comment:
Um zu verstehen, wie die Technologie funktioniert, gehen wir die Einrichtung von `https://example.com/` mit einem Zertifikatsverwaltungsagenten durch, der Let's Encrypt unterstützt.

Reply:
This is the existing translation without a change.

Reply:
This is actually the same issue as below with the double spaces.